boards.txt

```diff
@@ -20300,6 +20300,116 @@ department_of_alchemy_minimain_esp32s2.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+
+Bee_Data_Logger.name=Bee Data Logger
+Bee_Data_Logger.vid.0=0x303a
+Bee_Data_Logger.pid.0=815C
+
+Bee_Data_Logger.bootloader.tool=esptool_py
+Bee_Data_Logger.bootloader.tool.default=esptool_py
+
+Bee_Data_Logger.upload.tool=esptool_py
+Bee_Data_Logger.upload.tool.default=esptool_py
+Bee_Data_Logger.upload.tool.network=esp_ota
+
+Bee_Data_Logger.upload.maximum_size=1310720
+Bee_Data_Logger.upload.maximum_data_size=327680
+Bee_Data_Logger.upload.flags=
+Bee_Data_Logger.upload.extra_flags=
+Bee_Data_Logger.upload.use_1200bps_touch=true
+Bee_Data_Logger.upload.wait_for_upload_port=true
+Bee_Data_Logger.upload.speed=921600
+
+Bee_Data_Logger.serial.disableDTR=false
+Bee_Data_Logger.serial.disableRTS=false
+
+Bee_Data_Logger.build.tarch=xtensa
+Bee_Data_Logger.build.bootloader_addr=0x0
+Bee_Data_Logger.build.target=esp32s3
+Bee_Data_Logger.build.mcu=esp32s3
+Bee_Data_Logger.build.core=esp32
+Bee_Data_Logger.build.variant=Bee_Data_Logger
+Bee_Data_Logger.build.board=BeeMotionS3
```

Comment:
build.board should be all upper case and not mixed-case, this will be available at compilation time as `-DARDUINO_{build.board}` which expands to `-DARDUINO_BeeMotionS3` as the PR is currently.